CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ Updated SRA-Tools to 2.10.8, which improves stability of the downloading and unp
 
 ### `Added`
 
+## 2020-12-10
+
+- New process for automatic metadata download by [@SusiJo](https://github.com/SusiJo)
```

Comment:
```suggestion
- [#4](https://github.com/qbic-pipelines/sradownloader/pull/4) New process for automatic metadata download by [@SusiJo](https://github.com/SusiJo)
```

Reply:
Nice, you can also add a link to the exact PR to be even more precise on the changes, it is good practice :)